common/Unit.cpp

```diff
@@ -185,6 +206,12 @@ void UnitWSD::lookupTile(int part, int width, int height, int tilePosX, int tile
     }
 }
 
+UnitWSD& UnitWSD::get()
+{
+    assert(GlobalWSD && GlobalWSD->_type == UnitType::Wsd);
```

Comment:
If we guarantee that GlobalWSD always holds an instances of UnitWSD, why do we need to store the type and compare it?